app/models/notification.rb

```diff
@@ -51,6 +51,7 @@ class Notification < ActiveRecord::Base
         ->(user_id, limit: 30) do
           where(user_id: user_id).visible.prioritized.includes(:topic).limit(limit)
         end
+  scope :for_chat_mentions, -> { where(notification_type: types[:chat_mention]) }
```

Comment:
We don't want to have chat stuff in core

Reply:
Oh, right, a very good point, I'll make sure to correct it.